apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.ts

```diff
@@ -20,11 +21,12 @@ export class DotContainerContentletService {
      */
     getContentletToContainer(
         container: DotPageContainer,
-        content: DotPageContent
+        content: DotPageContent,
+        page: DotPage
```

Comment:
add new param in doc

Reply:
done https://github.com/dotCMS/core-web/pull/1803/commits/7fba5d5be372b9a2235170b8758f0d224ff56026#diff-262bf1d91bb39253a5abd06e62d539aede708d0efc4bfb4af4ca1d5b7a33b402R19